doc/mgr/administrator.rst

```diff
@@ -53,6 +53,27 @@ by a standby.
 If you want to pre-empt failover, you can explicitly mark a ceph-mgr
 daemon as failed using ``ceph mgr fail <mgr name>``.
 
+Performance
+-----------
+
+All the mgr modules share a cache that can be enabled with
+``ceph config set mgr mgr_ttl_cache_expire_seconds <seconds>``, where seconds
+is the time to live of the cached python objects.
+
+The default behaviour of a mgr daemon consists of sending python objects with
+data to the mgr modules. When you enable the cache the default python object
+formatter is replaced to a json serializator.
```

Comment:
Too technical for an end user? Perhaps this would make sense on devel docs (not sure if the ceph-mgr has devel-only docs).

Reply:
I was wondering about this too. I thought that maybe detailing that you are also changing the serialization process is relevant.